File: csharp/space-age/SpaceAge.cs
using System;

public class SpaceAge
{
    private readonly int _seconds;
    public double EarthYears { get; private set; }

    public SpaceAge(int seconds)
    {
        _seconds = seconds;
    }

    public double OnEarth() => _seconds / 31_557_600.0;

    public double OnMercury() => OnEarth() / 0.2_408_467;

    public double OnVenus() => OnEarth() / 0.61_519_726;

    public double OnMars() => OnEarth() / 1.8_808_158;

    public double OnJupiter() => OnEarth() / 11.8_626_15;

    public double OnSaturn() => OnEarth() / 29.447_498;

    public double OnUranus() => OnEarth() / 84.016_846;

    public double OnNeptune() => OnEarth() / 164.79_132;
}

File: java/space-age/src/main/java/SpaceAge.java
class SpaceAge {

    final double seconds;

    SpaceAge(double seconds) {
        this.seconds = seconds;
    }

    double getSeconds() {
        return seconds;
    }

    double onEarth() {
        return seconds / 31_557_600;
    }

    double onMercury() {
        return onEarth() / 0.2_408_467;
    }

    double onVenus() {
        return onEarth() / 0.61_519_726;
    }

    double onMars() {
        return onEarth() / 1.8_808_158;
    }

    double onJupiter() {
        return onEarth() / 11.8_626_15;
    }

    double onSaturn() {
        return onEarth() / 29.447_498;
    }

    double onUranus() {
        return onEarth() / 84.016_846;
    }

    double onNeptune() {
        return onEarth() / 164.79_132;
    }
}
